src/__tests__/widget_test.ts

```diff
@@ -139,23 +139,6 @@ widgets.forEach((widgetClass) => {
     })
 
     describe("Post Message Dispatching", () => {
-      test("message is dispatched to the appropriate callback", () => {
```

Comment:
This test was failing because it was triggering the `mx/load` message (which then caused the widget to dispatch a message back to the widget) before without mounting the iframe, which would cause the metrics post message to fail.

This test and the one below it test the same thing, so I'm just removing this one.